src/foundry/foundry.js/pixi/containers/placeableObject.d.ts

```diff
@@ -1,425 +1,477 @@
-  protected _getShiftedPosition(dx: number, dy: number): { x: number; y: number };
+    /**
+     * Callback actions which occur on a single right-click event to configure properties of the object
+     * @see MouseInteractionManager#_handleClickRight
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onClickRight(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Activate interactivity for the Placeable Object
-   */
-  activateListeners(): void;
+    /**
+     * Callback actions which occur on a double right-click event to configure properties of the object
+     * @see MouseInteractionManager#_handleClickRight2
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onClickRight2(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Create a standard MouseInteractionManager for the PlaceableObject
-   */
-  protected _createInteractionManager(): NonNullable<this['mouseInteractionManager']>;
+    /**
+     * Callback actions which occur when a mouse-drag action is first begun.
+     * @see MouseInteractionManager#_handleDragStart
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onDragLeftStart(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Actions that should be taken for this Placeable Object when a mouseover event occurs
-   * @param options - (default: `{}`)
-   * @param hoverOutOthers - (default: `true`)
-   */
-  protected _onHoverIn(event: PIXI.InteractionEvent, options?: { hoverOutOthers: boolean }): boolean | void;
+    /**
+     * Callback actions which occur on a mouse-move operation.
+     * @see MouseInteractionManager#_handleDragMove
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onDragLeftMove(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Actions that should be taken for this Placeable Object when a mouseout event occurs
-   */
-  protected _onHoverOut(event: PIXI.InteractionEvent): boolean | void;
+    /**
+     * Callback actions which occur on a mouse-move operation.
+     * @see MouseInteractionManager#_handleDragDrop
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onDragLeftDrop(event: PIXI.InteractionEvent): Promise<false | this | D[]>;
 
-  /**
-   * Callback actions which occur on a single left-click event to assume control of the object
-   */
-  protected _onClickLeft(event: PIXI.InteractionEvent): boolean | null;
+    /**
+     * Callback actions which occur on a mouse-move operation.
+     * @see MouseInteractionManager#_handleDragCancel
+     * @param event - The triggering mouse click event
+     */
+    protected _onDragLeftCancel(event: MouseEvent): void;
 
-  /**
-   * Callback actions which occur on a double left-click event to activate
-   */
-  protected _onClickLeft2(event: PIXI.InteractionEvent): void;
+    /**
+     * @deprecated since 0.8.0
+     * @param options - (default: `{}`)
+     */
+    static create(
```

Comment:
technically, `create` can be called both with a single element or an array. however, as it's deprecated, it doesn't matter that much, I suppose... But the correct type for the `data` parameter would probably be
```ts
Parameters<InstanceType<ConfiguredDocumentClass<typeof Scene>>['createEmbeddedDocuments']>[0] | Parameters<InstanceType<ConfiguredDocumentClass<typeof Scene>>['createEmbeddedDocuments']>[0][number]
```
or so